src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+function printMessage(messageText, messageClass) {
+  const message = document.createElement('div');
+
+  message.classList.add(messageClass);
+  message.innerText = messageText;
+
+  document.body.append(message);
+}
+
+const promise1 = new Promise((resolve) => {
+  const logoButton = document.querySelector('.logo');
+
+  logoButton.addEventListener('click', () => {
+    printMessage('Promise was resolved!', 'message');
+    resolve();
+  });
+});
+
+promise1
```

Comment:
```suggestion
promise1
```
Це має бути новий проміс, а не той же.
Створи два різних проміса

Reply:
Спробував переробити